Makefile

```diff
@@ -88,6 +88,12 @@ LOG=ANSIBLE_LOG_PATH="$(LOGFILE)"
 LOGINFO=$(INFO) "Wrote $(LOGFILE)"
 endif
 
+ifndef ANSIBLE_COLLECTIONS_PATH
```

Comment:
I think setting ANSIBLE_COLLECTIONS_PATH in the makefile targets should be unconditional. We always want to use the local collections in the collections directory. The fact we were not was a bug.